hyp3_autorift/process.py

```diff
@@ -66,7 +67,7 @@ def get_lc2_path(metadata):
     if metadata['id'][3] in ('4', '5'):
         band = metadata['assets'].get('B2.TIF')
         if band is None:
-            band = metadata['assets']['red']
+            band = metadata['assets']['green']
```

Comment:
@jhkennedy and @jacquelynsmale, I changed this line to green because Band 2 (the band Alex suggested we use) in [Landsat 4/5 is green](https://www.usgs.gov/landsat-missions/landsat-5). Let me know if this isn't correct.